dev/hal_install_spin.docker

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:14.04
+# Parameters of the GCP project Spinnaker is maintaining.
+ARG BUILD_PROJECT
+ARG BUILD_ACCOUNT
+ARG BUILD_KEY # json keyfile contents
```

Comment:
Secrets shouldn't be build-time args, since that would have us baking keys into images.